### PR TITLE
Jenkification.

### PR DESCRIPTION
**Pull request by ehiggs**

### CODE REVIEW

File: test/unit/commands/test_commands.py [hunk 1]
```diff
@@ -31,11 +31,11 @@
...
-    @unittest.expectedFailure
-    def test_command_breathing(self):
-        '''test command can actually be created.'''
-        c = hcc.Command()
-        self.assertEqual(str(c), '')
+    #@unittest.expectedFailure
+    #def test_command_breathing(self):
+    #    '''test command can actually be created.'''
+    #    c = hcc.Command()
+    #    self.assertEqual(str(c), '')
```
Comment (ehiggs) [inline test/unit/commands/test_commands.py]: python 2.6 doesn't support expectedFailure.

File: test/unit/test_mpiservice.py [hunk 2]
```diff
@@ -88,9 +88,9 @@ def test_slave_spread(self):
         tasks = hm._slave_spread(ms.comm)
         self.assertEqual(tasks, None)
 
-    @unittest.expectedFailure
-    def test_mpiservice_run_dist(self):
-        '''test mpiservice run dist'''
-        ms = hm.MpiService()
-        ms.distribution()
-        ms. run_dist()
+    #@unittest.expectedFailure
```
Comment (boegel) [inline test/unit/test_mpiservice.py]: this?

File: test/unit/rmscheduler/test_resourcemanagerscheduler.py [hunk 3]
```diff
@@ -37,11 +37,11 @@ def test_resourcemanagerscheduler_init(self):
         self.assertTrue('cwd' in o.vars)
         self.assertTrue('jobid' in o.vars)
 
-    @unittest.expectedFailure
-    def test_resourcemanagerscheduler_submit(self):
-        '''test ResourceManagerScheduler submit'''
-        o = hrr.ResourceManagerScheduler(None)
-        o.submit() # TODO: Remove.
+    #@unittest.expectedFailure
```
Comment (boegel) [inline test/unit/rmscheduler/test_resourcemanagerscheduler.py]: this?

File: setup.py [hunk 4]
```diff
@@ -33,9 +33,6 @@
 import subprocess
 from setuptools import setup, Command
 
-if sys.version_info[:2] < (2, 7):
-    raise RuntimeError("Python version 2.7 required.")
```
Comment (boegel) [inline setup.py]: better keep a check like this in?

py2.6 or newer, no py3?

File: .coveragerc [hunk 5]
```diff
@@ -1,4 +1,5 @@
 [run]
+#branch=True
```
Comment (boegel) [inline .coveragerc]: why is this commented out?

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Jenkins: test this please

File: setup.py (hunk 4 above)
Comment (ehiggs) [inline setup.py]: This is documented as a classifier now.

**Pull request merged**